client/reader/search-stream/site-results.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External Dependencies
+ */
+import React, { PropTypes } from 'react';
+import 'lodash';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { getReaderFeedsForQuery } from 'state/selectors';
+import QueryReaderFeedsSearch from 'components/data/query-reader-feeds-search';
+import SubscriptionListItem from 'blocks/reader-subscription-list-item/connected';
+
+class SitesResults extends React.Component {
+	static propTypes = {
+		query: PropTypes.string,
+	};
+
+	render() {
+		const { query, translate, searchResults } = this.props;
+
+		return (
+			<div>
+				<QueryReaderFeedsSearch query={ query } />
+				{ searchResults
+					? searchResults.map(
+							feed => (
+								<SubscriptionListItem
+									key={ feed.feed_ID }
+									feedId={ +feed.feed_ID }
+									blogId={ +feed.blogId }
+									showLastUpdatedDate={ false }
+								/>
+							)
+						)
+					: translate( 'no results' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'No Results' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/site-search/headers-in-search)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).